rust/build.sh

```diff
@@ -19,4 +19,4 @@ if [[ -z $TAG ]]; then
   fi
 fi
 
-DOCKER_BUILDKIT=1 docker build $PLATFORM -t gcr.io/abacus-labs-dev/hyperlane-agent:$TAG .
+DOCKER_BUILDKIT=1 docker build -f rust/Dockerfile $PLATFORM -t gcr.io/abacus-labs-dev/hyperlane-agent:$TAG .
```

Comment:
instead of changing this by hardcoding `-f rust/Dockerfile`, add the following after the shebang at the top
```sh
# Change directory to where script is
cd "$(dirname "$0")"
```

so you'll be able to run rust/build.sh from whatever path you're at